api/notifications/notifications.py

```diff
@@ -3,28 +3,33 @@
 from notifications.utils import gen_subject_and_message
 from actions.models import ActionPriority, ActionStatus
 from django.core.mail import EmailMessage
-from users.models import Volunteer
 from django.utils import timezone
 import logging
 import os
 
 
+coordinator_email = os.getenv("COORDINATOR_EMAIL", "coordinators@test.com")
 from_email = os.getenv("EMAIL_HOST_USER", "test@test.com")
+site_name = os.getenv("ADMIN_SITE_TITLE", "TestTitle")
 site_url = os.getenv("SITE_URL", "http://0.0.0.0:80")
 logger = logging.getLogger(__name__)
 
 
-def send_invite(user):
-    """Sends an email invite to a User instance,
+def send_invite(person):
```

Comment:
Relying on `User` allowed not to have to check if the `person` has an account or not. Creating a `Person` creates a `User` which triggered the notification. It also allowed to keep all things account related bound to Django's admin model rather than mix with our own.

Reply:
Wouldn't that also trigger an invite email when creating a Coordinator that already has a Volunteer account (and vice-versa)? At that point, the user already has an account so shouldn't receive anything.